### PR TITLE
Object code for calendar creation

### PR DESCRIPTION
**Pull request by derachimso**

### CODE REVIEW

File: capstone/src/main/java/servlet/Event.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+public class Event{
+    String id; 
+    String location;
+    String summary;
+
+    public void EventSummary(String summary, String id) {
+        this.summary = summary;
+        this.id = id;
+
```
Comment (magwitz) [inline capstone/src/main/java/servlet/Event.java:9]: Delete line

File: capstone/src/main/java/servlet/Event.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    }
+
+     public void Event(String summary, String id, String location) {
+        this.summary = summary;
+        this.id = id;
+        this.location = location;
+
```
Comment (magwitz) [inline capstone/src/main/java/servlet/Event.java:16]: delete line

File: capstone/src/main/java/servlet/Event.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+public class Event{
```
Comment (magwitz) [inline capstone/src/main/java/servlet/Event.java:1]: Does this need setters as well? Or do you anticipate only setting the fields when you first create the object?

File: capstone/src/main/java/servlet/Event.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    public String getid() {
+        return id;
+    }
+    public String getlocation() {
+        return location;
+    }
+    public String getsummary() {
+        return summary;
+    }
+
```
Comment (magwitz) [inline capstone/src/main/java/servlet/Event.java:27]: delete line

File: capstone/src/main/java/servlet/Event.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        this.id = id;
+
+    }
+
+     public void Event(String summary, String id, String location) {
+        this.summary = summary;
+        this.id = id;
+        this.location = location;
+
+    }
```
Comment (magwitz) [inline capstone/src/main/java/servlet/Event.java:17]: for consistency, let's add an empty line after each of the methods.

**Comment by magwitz** [comment]
Another comment. Let's make a folder called "data" where we put this instead of putting it in "servlets" since it isn't a servlet.

File: capstone/src/main/java/servlet/Event.java (hunk 3 above)
Comment (magwitz) [inline capstone/src/main/java/servlet/Event.java:1]: Add a package declaration at the top of this file:

`package com.google.sps.data;`

File: capstone/src/main/java/servlet/Event.java (hunk 4 above)
Comment (derachimso) [reply capstone/src/main/java/servlet/Event.java:27]: I'm not clear on which line i should delete. Do you mean this block of code or line 27

File: capstone/src/main/java/servlet/Event.java (hunk 2 above)
Comment (derachimso) [reply capstone/src/main/java/servlet/Event.java:16]: I'm also not clear on which line i should delete. Do you mean this block of code or line 16

File: capstone/src/main/java/servlet/Event.java (hunk 1 above)
Comment (derachimso) [reply capstone/src/main/java/servlet/Event.java:9]: I'm still not clear on which line i should delete.

Comment (magwitz) [reply capstone/src/main/java/servlet/Event.java:9]: Line 9 -- it's just unnecessary whitespace

File: capstone/src/main/java/servlet/Event.java (hunk 2 above)
Comment (magwitz) [reply capstone/src/main/java/servlet/Event.java:16]: Line 16

File: capstone/src/main/java/servlet/Event.java (hunk 4 above)
Comment (magwitz) [reply capstone/src/main/java/servlet/Event.java:27]: Line 27

File: capstone/src/main/java/servlet/Event.java (hunk 3 above)
Comment (derachimso) [reply capstone/src/main/java/servlet/Event.java:1]: Silas said he needed only getters

**Pull request merged**